portal/import_template.php

```diff
@@ -271,190 +285,211 @@ function renderProfileHtml()
-                            $this_name = attr($file['template_name']);
-                            if ($file['mime'] === 'application/pdf') {
-                                continue;
+                <div class='col-7 col-height'>
+                    <div id="edit-profiles" class='control-group mx-1 border-left border-right'>
+                        <?php
+                        foreach ($profile_list as $profile => $profiles) {
+                            $profile_items_list = $templateService->getTemplateListByProfile($profile);
+                            $profile_esc = attr($profile);
+                            $events = $templateService->fetchAllProfileEvents();
+                            $recurring = attr($events[$profile]['recurring'] ?? '');
+                            $trigger = attr($events[$profile]['event_trigger'] ?? '');
+                            $days = attr($events[$profile]['period'] ?? '');
+                            ?>
+                            <form id="<?php echo $profile_esc ?>-form" name="<?php echo $profile_esc; ?>" class='form form-inline bg-dark text-light py-1 pl-1'>
+                                <label class='mr-1'><?php echo xlt($profiles['title']) ?></label>
+                                <div class='input-group-prepend'>
+                                    <label for="<?php echo $profile_esc ?>-recurring" class="form-check-inline"><?php echo xlt('Recurring') ?>
+                                        <input <?php echo $recurring ? 'checked' : '' ?> name="recurring" type='checkbox' class="input-control ml-1 mt-1" id="<?php echo $profile_esc ?>-recurring" />
+                                    </label>
+                                </div>
+                                <div class='input-group-prepend'>
+                                    <label for="<?php echo $profile_esc ?>-when"><?php echo xlt('On') ?></label>
+                                    <select name="when" class='input-control-sm mx-1' id="<?php echo $profile_esc ?>-when">
+                                        <option value="completed"><?php echo xlt('Completed') ?></option>
+                                        <option <?php echo $trigger === 'always' ? 'selected' : ''; ?> value='always'><?php echo xlt('Always') ?></option>
+                                        <option <?php echo $trigger === 'once' ? 'selected' : ''; ?> value='once'><?php echo xlt('One time') ?></option>
+                                    </select>
+                                </div>
+                                <div class='input-group-prepend'>
+                                    <label for="<?php echo $profile_esc ?>-days"><?php echo xlt('Every') ?></label>
+                                    <input name="days" type="text" style="width: 50px" class='input-control-sm ml-1' id="<?php echo $profile_esc ?>-days" placeholder="<?php echo xlt('days') ?>" value="<?php echo $days ?>" />
```

Comment:
xlt in above line should be xla
(hopefully the ci testing has not completely broken; will play around with it this evening)